core/templates/pages/exploration-editor-page/services/change-list.service.spec.ts

```diff
@@ -322,6 +326,27 @@ describe('Change List Service when changes are mergable', () => {
       state_name: 'Introduction'
     });
   });
+
+  it('should update skill editor browser tabs unsaved changes status', () => {
+    let explorationEditorBrowserTabsInfo = EntityEditorBrowserTabsInfo.create(
+      'exploration', 'exp_id', 2, 1, false);
+    spyOn(
+      localStorageService, 'getEntityEditorBrowserTabsInfo'
+    ).and.returnValue(explorationEditorBrowserTabsInfo);
+    spyOn(
+      localStorageService, 'updateEntityEditorBrowserTabsInfo'
+    ).and.callFake(() => {});
+
+    expect(
+      explorationEditorBrowserTabsInfo.doesSomeTabHaveUnsavedChanges()
+    ).toBeFalse();
+
+    changeListService.addState('state');
+
+    expect(
+      explorationEditorBrowserTabsInfo.doesSomeTabHaveUnsavedChanges()
+    ).toBeTrue();
```

Comment:
Please add comments explaining when it should be true and false, Or you can split these into 2 individual tests.